SUMMARY.md

```diff
@@ -1 +1,2 @@
 * [Pony for C#](csharp.asciidoc)
+* [Pony for Java](java.asciidoc)
```

Comment:
Carl, do you think it would make sense in the TOC to have links into subsections or no?

I'm of two minds about this.